server/tidb_test.go

```diff
@@ -136,10 +133,13 @@ func (ts *tidbTestSuite) TestPreparedTimestamp(c *C) {
 // this test will change `kv.TxnTotalSizeLimit` which may affect other test suites,
 // so we must make it running in serial.
 func (ts *tidbTestSerialSuite) TestLoadData(c *C) {
-	c.Parallel()
```

Comment:
Why remove this code？

Reply:
`.Parallel()` will run multi test cases run in parallel and test cases in this suite should be run in serial. In my test, this parallel will cause test `TestStmtCount` failed because  the statements running by this case will also contribute to server metrics thus make statement metrics counting wrong